.github/workflows/on_pull_request.yml

```diff
@@ -1,9 +1,12 @@
 name: PR Checks
 
-on: pull_request
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
```

Comment:
Should we move `ready_to_review` into another file so that we only run the jobs that weren't ran on draft?

Reply:
I can try it. `ready_for_review` is a special event. My worry is, what I tend to do it push new code or a rebase, and then make it ready for review, and I'm not sure if that would cancel all the other jobs that are legit needed. But I won't know until I try it.

Reply:
OK, so this won't necessarily work either. Mainly because you end up needing two different named checks for Chromatic because it has to live in its own workflow file with its own name. 

`ready_for_review` only ever fires  when you hit that button, so you need a chromatic job for `opened, synchronize, reopened` which are the default events for every other thing you do in a PR.